### PR TITLE
ci: Fix failing CI

### PR DESCRIPTION
**Pull request by siketyan**

### CODE REVIEW

File: .fvm/fvm_config.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "flutterSdkVersion": "3.18.0-0.2.pre",
+  "flutterSdkVersion": "3.18.0-0.2.pre@beta",
```
Comment (siketyan) [inline .fvm/fvm_config.json:2]: flutter-fvm-config-action requires `@<channel>` annotation to determine which release channel to use: https://github.com/kuhnroyal/flutter-fvm-config-action/blob/34c3905bc939a4ff9d9cb07d5a977493fa73b2aa/index.js#L50-L52

**Comment by siketyan** [comment]
@YumNumm Could you please review the changes and approve to run the workflow?

**Review by YumNumm — APPROVED** [review]
LGTM!

**Comment by siketyan** [comment]
Seems the project have some private dependencies?

image

**Comment by siketyan** [comment]
gimme a sec to fix it

**Comment by YumNumm** [comment]
Thank you for contributing🙏

I'm out now.
I'll have a look at it in a couple of hours.

**Comment by siketyan** [comment]
Still WIP 🙏  Re-updating pubspec.lock using the beta SDK

**Comment by siketyan** [comment]
@YumNumm Done. Could you please review the changes when you have time?

File: pubspec.lock [hunk 2]
```diff
@@ -934,26 +944,28 @@ packages:
...
     dependency: "direct main"
     description:
-      path: "../flutter-maplibre-gl"
-      relative: true
-    source: path
+      path: "."
+      ref: "0bf326044759dfc2f590a1c219d2f1eb86e7b8ad"
+      resolved-ref: "0bf326044759dfc2f590a1c219d2f1eb86e7b8ad"
+      url: "https://github.com/YumNumm/flutter-maplibre-gl.git"
+    source: git
```
Comment (siketyan) [inline pubspec.lock:941]: Seems pubspec.lock was outdated (overriden somewhere?)

**Comment by siketyan** [comment]
Due to GitHub's limitation, we cannot grant write permission to GitHub Actions when it is run by a pull_request event from a forked repo. To mitigate that, we need to change the trigger to pull_request_target.

See: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

**Comment by siketyan** [comment]
Attempted to change the trigger at [b69d52d](https://github.com/YumNumm/EQMonitor/pull/502/commits/b69d52da1c8817a1c7691952185eb9d24d1496ca). However the change will be applied after this PR is merged.

File: pubspec.lock (hunk 2 above)
Comment (YumNumm) [reply pubspec.lock:941]: I forgot to add `pubspec_overrides.yaml`. 

* added `pubspec_overrides.yaml` at 9a03428
* executed `flutter pub get` at ea5ff41

**Comment by YumNumm** [comment]
then, let me check CI in develop branch :)

**Pull request merged**